tests/integration/conftest.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Canonical Ltd.
+# Licensed under the GPLv3, see LICENCE file for details.
+
+import configparser
+import re
+
+import pytest
+import pytest_asyncio
+import pytest_operator.plugin
+
+
+@pytest.fixture
+def openstack_environment(request):
+    """Parse the openstack rc style configuration file from the --openstack-rc argument
+
+    Return a dictionary of environment variables and values.
+    """
+    rc_file = request.config.getoption("--openstack-rc")
```

Comment:
What if the `--openstack-rc` parameter was not passed?

Reply:
see above

Reply:
I realise this has been merged, would it be worth a assert/ check and give feedback to the user if it is not provided?